spec/features/repositories_spec.rb

```diff
@@ -49,6 +49,8 @@
       expectations = [["tag0"], ["tag1"], ["tag2", "tag3"], ["tag4"], ["tag5"]]
 
       visit repository_path(repository)
+      sleep 1
```

Comment:
We probably don't need this. This is a leftover of me trying to debug a test failure that happens randomly (not caused by this commit, but by some unrelated previous one).